src/pages/core/Profile/index.js

```diff
@@ -75,21 +80,11 @@ class Profile extends Component {
         body[updatedAttrs[i]] = user[updatedAttrs[i]]
       }
 
-      fetch(`${ BACKEND_URL }data/user/${ getUserID() }`, {
-        method: 'PATCH',
-        headers: authHeader(),
-        mode: 'cors',
-        credentials: 'omit',
-        body: JSON.stringify(body),
-      })
-        .then(response => {
-          if(!response.ok) throw new Error(response)
-          else return response.json()
-        })
-        .then(data => {
-          this.setState({success: true})
-          setUserProfile(body)
-        })
+      this.props.updateUser({
+        id: getUserID(),
+        patch: body
+      });
```

Comment:
No error handling for this update seems to be set up. Or, have I just overlooked it? How is the component notified that the update has failed?